docs/features.rst

```diff
@@ -81,12 +81,12 @@ General features
      - API not available.
    * - `Transport Plugins <https://community.rti.com/static/documentation/connext-dds/6.0.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/index.htm#UsersManual/transports.htm>`__
      - Partial
-     - The built-in transports can be configured in XML, but add-ons cannot be loaded (see next).
+     - The built-in transports can be configured in XML.
    * - Add-on Libraries 
        (such as `Monitoring <https://community.rti.com/static/documentation/connext-dds/6.0.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/index.htm#UsersManual/PartMonitoringLib.htm>`__, 
        `Security Plugins <https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds/dds_security/html_files/RTI_SecurityPlugins_GettingStarted/index.htm>`__ )
-     - Not supported
-     - *Connector* currently cannot load dynamically linked add-on libraries.
+     - Supported
+     - See :ref:`Loading Connext DDS Add-On Libraries`.
 
```

Comment:
The links to the docs above are not right, but that is being handled in https://github.com/rticommunity/rticonnextdds-connector-py/pull/51 (and https://github.com/rticommunity/rticonnextdds-connector-js/pull/28).